wordpress/wp-content/plugins/cds-base/languages/cds-snc-fr_CA.po

```diff
@@ -464,24 +449,22 @@ msgstr "Aucune liste trouvée"
 
 #: classes/Modules/Notify/FormHelpers.php:79
 msgid "Send template to list"
-msgstr "Envoyer le gabarit à la liste"
+msgstr "Envoyer le modèle à la liste"
 
 #: classes/Modules/Notify/ListManagerSettings.php:48
 #: classes/Modules/Notify/ListManagerSettings.php:49
-#, fuzzy
 msgid "Lists"
-msgstr "ID de liste"
+msgstr ""
 
 #: classes/Modules/Notify/ListManagerSettings.php:62
-#, fuzzy
 msgid "Your Lists"
-msgstr "Votre message"
+msgstr ""
 
 #: classes/Modules/Notify/ListManagerSettings.php:67
 #: classes/Modules/Notify/ListManagerSettings.php:76
 #: classes/Modules/Notify/ListManagerUserProfile.php:37
 msgid "List Manager"
-msgstr "Gestionnaire des listes"
+msgstr "Gestionnaire de liste"
```

Comment:
I think the old translation was right. The old translation was like “Manager for lists”, and the new one is like “Manager for the list”